### DISCUSSION TITLE
Which documentation/topics could be best improved?

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#27787] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### Please tell us more about your question or problem

Which Renovate topics/capabilities do you think could benefit the most from improved documentation? Which feature took you the longest to figure out?

e.g.
- automerge
- versioning
- admin vs repository config
- config preset

Please suggest one topic per comment below (it's ok to add multiple, but maybe keep to 3 per person max to not flood things) and then I'd like to see how many thumbs-up each comment gets. I'll try to set aside time to add documentation for those you think are the most pain points.

**Comment by HonkingGoose** [comment]
It's probably not worth creating a new discussion category for questions like these? I'm asking because GitHub Discussions allows proper polls:

https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-poll

**Reply by rarkins** [reply]
I would like this to be more open ended than a poll
